CONTRIBUTING.md

```diff
@@ -2,26 +2,26 @@
   Thx for your interest! We're so glad you're here. 
 
 ### Important Resources
-  - docs: [https://github.com/packethost/standards/wiki](https://github.com/packethost/standards/wiki)
```

Comment:
I'm going to remove the link to the wiki and consider embedding similar text in the README.md or elsewhere.  This text may be similar enough to what is provided in the purpose-and-governance.md file that it does not need to be persisted.

I would like to disable the wiki feature when we move this repo to equinix-labs/equinix-labs.

Here is the wiki text in its current form.  This is the only page in the wiki:

```
About https://github.com/packethost/standards Packet maintains a number of public repositories that help customers to run various workloads on Packet. These repositories are in various states of completeness and quality, and being public, developers often find them and start using them. This creates problems:

Developers using low-quality repositories may infer that Packet generally provides a low quality experience. Many of our repositories are put online with no formal communication with, or training for, customer success. This leads to a below average support experience when things do go wrong. We spend a huge amount of time supporting users through various channels when with better upfront planning, documentation and testing much of this support work could be eliminated. To that end, we propose three tiers of repositories: Private, Experimental, and Maintained.

The Goal Our repositories should be the example from which adjacent, competing, projects look for inspiration.

Each repository should not look entirely different from other repositories in the ecosystem, having a different layout, a different testing model, or a different logging model, for example, without reason or recommendation from the subject matter experts from the community.

We should share our improvements with each ecosystem while seeking and respecting the feedback of these communities.

Whether or not strict guidelines have been provided for the project type, our repositories should ensure that the same components are offered across the board. How these components are provided may vary, based on the conventions of the project type. GitHub provides general guidance on this which they have integrated into their user experience.
```

Reply:
Do we need to preserve this text @alice-sowerby ?

Reply:
No you can delete that